main.py

```diff
@@ -921,7 +875,7 @@ def GetPlayGroundBikeRideKey(self, promoID):
         )
 
         response = self.HttpRequest(url, headers, "POST", 200, payload)
-        if response is None:
+        if response is None or not response:
```

Comment:
According to this commit: https://github.com/masterking32/MasterHamsterKombatBot/commit/820f0b7d938b62578e4bdc597ebe4bd78464e54f , these changes are not needed anymore!